Misc/NEWS.d/next/Library/2020-03-08-09-53-55.bpo-36287.mxr5m8.rst

```diff
@@ -0,0 +1,3 @@
+:func:`ast.dump()` no longer output optional fields and attributes. Values
```

Comment:
```suggestion
:func:`ast.dump()` no longer outputs optional fields and attributes with default values. Default values
```

Reply:
I think this explanation is slightly confusing. If I understand correctly is the **default** values what are now class attributes. For instance, for a non-default "kind":

```
>>> x = ast.parse("u'dsf'", mode="eval")
>>> x.body.kind
'u'
```

while the default for `kind` is what is set on the class:

```
>>> print(type(x.body).kind)
None
```